README.md

```diff
@@ -65,9 +66,13 @@ You may want to configure the binary to run as a service so it can start on syst
 
 ### <a name="docker"></a> Docker
 
-Run the Docker image directly from the OutSystems GitHub Container registry:
+Run the Docker image directly from the OutSystems GitHub container registry:
 
-    docker run --rm -it ghcr.io/outsystems/outsystemscc:latest --help
+    docker run --rm -it ghcr.io/outsystems/outsystemscc --help
+
+If you're running the container on a runtime where you need to specify the command line or override the entrypoint (for example on Azure Container Instances or AWS Fargate):
+
+    docker run --rm -it --entrypoint /app/outsystems ghcr.io/outsystems/outsystemscc --help
```

Comment:
Shouldn't be `--entrypoint /app/outsystemscc` ?
In PR: https://github.com/OutSystems/cloud-connector/pull/21/files the filename is `outsystemscc`

Reply:
Correct, please fix also the doc. Thank you